website/src/components/AssemblyGuideImage.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025 Enactic, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, {
+  type ReactNode
+} from 'react';
+import BlockImage, { type BlockImageProps } from './BlockImage';
+
+export default function AssemblyGuideImage({ src, alt, width }: BlockImageProps): ReactNode {
+  return (
+    <BlockImage
+      src={`hardware/assembly-guide/${src}`}
+      alt={alt}
+      width={width}
+    />
+  );
+}
```

Comment:
It looks like AssemblyGuideImage is just a thin wrapper around BlockImage and doesn’t add any extra logic. We could simplify things by removing this component altogether and using BlockImage directly

Reply:
Sure.